src/cli/utils.js

```diff
@@ -12,6 +12,16 @@ function existsStat() {
 	}
 }
 
+
+function getIgnoreList( baseDir ) {
+	if ( fs.existsSync( `${baseDir}/.gitignore` ) ) {
+		let gitIgnore = fs.readFileSync( `${baseDir}/.gitignore`, "utf-8" );
+		gitIgnore = gitIgnore.trim();
+		return gitIgnore.split( "\n" );
```

Comment:
Can we avoid the reassignment by just doing:

```javascript
const gitIgnore = fs.readFileSync( `${baseDir}/.gitignore`, "utf-8" );
return gitIgnore.trim().split( "\n" );
```

Reply:
I will make this change.